### ISSUE TITLE
USB install should be slightly more verbose

### ISSUE THREAD

**Issue by dorkmatt** [opencomputeproject/onie#214]
It would be useful if ONIE was slightly more verbose when attempting to mount a USB stick. In the current form, only a successful mount is logged as.. 

```
ONIE: Failed to configure eth0 interface
ONIE: Starting ONIE Service Discovery
ONIE: Executing installer: file://dev/sda1/onie-installer-powerpc
Verifying image checksum ... OK.
Preparing image archive ... OK.
```

However, if the stick is formatted incorrectly (wrong partition type) and/or has the wrong file name, the log doesn't show this as an error. Seems if we have a USB stick present and had issues with it, we should advertise that. Just as we do with eth0 not having link, a DHCP server not being found, etc.

**Comment by cobracmder** [comment]
Hey @dorkmatt , this sounds like a good idea.  Can you attach a patch/pr to get a glimpse of how much logging should be done?

**Comment by cbrune** [comment]
Just because a USB is present does not mean the user intends to boot from it.  It might be there for various reasons.  Today ONIE does not know it is USB, just that block device partitions are present.

That said we could print some messages about each partition found.  A mock up of the desired output would be a good starting point.

**Comment by cbrune** [comment]
This has been duplicated by issue #467.  A pull request for the fix is in #501.
